samples/EcsPipeline/main.cpp

```diff
@@ -14,6 +14,12 @@ int main()
     NovelRT::LoggingService logger = NovelRT::LoggingService();
     logger.setLogLevel(NovelRT::LogLevel::Info);
 
+#if NOVELRT_MOLTENVK_VENDORED
+    auto icdPath = NovelRT::Utilities::Misc::getExecutablePath() / "MoltenVK_icd.json";
+    setenv("VK_ICD_FILENAMES", icdPath.c_str(), 0);
+    logger.logInfo("macOS detected - setting VK_ICD_FILENAMES to path: {}", icdPath.c_str());
+#endif
```

Comment:
Is this just for the samples? Or will everyone who uses vendored MoltenVK have to do this?

Reply:
At the moment, it's just for the samples. There's future installation work that should resolve this for people using NovelRT outside of building the engine from source, however it was not covered here since our install targets are butchered atm (so I figured I'd do a mass cleanup of _all_ deps once our migration is over).